lib/core_ext/time_extensions.rb

```diff
@@ -0,0 +1,9 @@
+module TimeExtensions
```

Comment:
I think to follow conventions [1] this file should be called `lib/core_extensions/time/to_string.rb` and namespaced within a `CoreExtensions` module.

[1]: https://www.justinweiss.com/articles/3-ways-to-monkey-patch-without-making-a-mess/

Reply:
Ah, I was looking for that convention - remembered the core_extensions thing - thanks, I'll update

Reply:
Sorry @leenagupte - `to_string` - should that be `to_s`?

Reply:
Umm, not sure these will work - they are rails conventions, based on the class load path. But this is a Sinatra app - so plain 'ol Ruby.

Reply:
We've taken this discussion to another platform.